extensions/interactions/CodeRepl/CodeRepl.js

```diff
@@ -100,16 +114,23 @@ oppia.directive('oppiaInteractiveCodeRepl', [
           });
         };
 
+        var fixLineNumbers = function(err) {
+          var preCodeNumLines = $scope.preCode.split('\n').length;
+          return err.replace(/on line ([0-9]+)/g, function(match, p1) {
+            return '(on line ' + String(Number(p1) - preCodeNumLines) + ')';
+          });
+        };
+
```

Comment:
Thanks! I added the check, and now show different error messages depending on where the error is. Could you PTAL?